File: src/main/java/com/tripplanner/api/dto/MakeDto.java
package com.tripplanner.api.dto;

import java.util.List;

public class MakeDto {
	public MakeDto() { }

	public MakeDto(String name, List<ModelDto> models) {
		this.name = name;
		this.models = models;
	}

	private String name = "";
	private List<ModelDto> models;

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public List<ModelDto> getModels() {
		return models;
	}
}

File: src/main/kotlin/com/tripplanner/api/dto/MakeDto.kt
package com.tripplanner.api.dto

import com.tripplanner.api.entity.Make

class MakeDto {
	var name = ""
	lateinit var models: List<ModelDto>
		private set

	constructor() { }
	constructor(name: String, models: List<ModelDto>) {
		this.name = name
		this.models = models
	}

	constructor(make: Make) {
		this.name = make.getName()
		this.models = make.getModels().map { ModelDto(it) }
	}

}